helper_functions.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+function restartJudge {
```

Comment:
You should consider to rename these functions, for example `restartServer`, as these are related to the server but not to the complete judge.